scripts/lib/mkcloud-libvirt.sh

```diff
@@ -349,6 +349,12 @@ function libvirt_do_setuplonelynodes()
         local lonely_disk
         lonely_disk="$vdisk_dir/${cloud}.node$i"
 
+        lvdisplay $lonely_disk
+        if [ $? -ne 0 ]; then
+            hdd_size=${lonelynode_hdd_size}
+            _lvcreate ${cloud}.node$i $hdd_size $cloudvg
```

Comment:
it normally should have been created in the prepare step via libvirt_do_create_cloud_lvm
but this might still be useful to do here, when you want to add nodes later. Or we move it here altogether, so that volumes are not created before needed (same for normal cloud nodes)